src/store/global/transactions-store/types.ts

```diff
@@ -12,6 +12,10 @@ export interface TransactionsStoreState {
 export type ContainerState = TransactionsStoreState;
 
 export enum TxType {
+  STAKE = 'stake',
+  WITHDRAW = 'withdraw',
+  EXTEND = 'extend',
+  DELEGATE = 'delegate',
```

Comment:
Lets add STAKING_ prefix to prevent duplicates and make sure other devs will use correct type

```suggestion
  STAKING_STAKE = 'staking-stake',
  STAKING_WITHDRAW = 'staking-withdraw',
  STAKING_EXTEND = 'staking-extend',
  STAKING_DELEGATE = 'staking-delegate',
```

Reply:
done